components/brave_shields/browser/ad_block_regional_service_manager.cc

```diff
@@ -205,6 +205,35 @@ void AdBlockRegionalServiceManager::AddResources(const std::string& resources) {
   }
 }
 
+bool AdBlockRegionalServiceManager::IsFilterListAvailable(
+    const std::string& uuid) const {
+  DCHECK(!uuid.empty());
+  auto catalog_entry =
+      brave_shields::FindAdBlockFilterListByUUID(regional_catalog_, uuid);
+  return catalog_entry != regional_catalog_.end();
+}
+
+bool AdBlockRegionalServiceManager::IsFilterListEnabled(
+    const std::string& uuid) const {
+  DCHECK(!uuid.empty());
```

Comment:
this should have `DCHECK_CURRENTLY_ON(content::BrowserThread::UI);` since it's accessing local_state

Reply:
Done.